README.md

```diff
@@ -166,10 +172,7 @@ If you like the project, give it a star ⭐️
 
```

Comment:
**[OPTIONAL]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍